packages/webpack/src/utils/postcss-v8.js

```diff
@@ -0,0 +1,208 @@
+import fs from 'fs'
```

Comment:
Most code is duplicate of `postcss.js`, this is more handy when we fully migrate to v8: just remove `postcss.js` and rename `postcss-v8.js`